shared/AppInsightsCommon/src/Telemetry/Common/DataSanitizer.ts

```diff
@@ -134,7 +134,7 @@ export class DataSanitizer {
             const tempProps = {};
             for (let prop in properties) {
                 let value = properties[prop];
-                if (typeof value === "object" && typeof JSON !== "undefined") {
+                if (CoreUtils.isObject(value) && !CoreUtils.isUndefined(JSON)) {
```

Comment:
Will using `JSON` in this way crash where it does not exist? e.g. need some sort of `_getJSON` utility, or just keep the typeof check here.

Reply:
Yes, we should have a helper for this of continue to use the typeof JSON -- nice catch